patches/minecraft/net/minecraft/world/entity/projectile/LlamaSpit.java.patch

```diff
@@ -1,10 +1,20 @@
 --- a/net/minecraft/world/entity/projectile/LlamaSpit.java
 +++ b/net/minecraft/world/entity/projectile/LlamaSpit.java
-@@ -29,6 +_,7 @@
+@@ -29,7 +_,16 @@
        super.m_8119_();
        Vec3 vec3 = this.m_20184_();
        HitResult hitresult = ProjectileUtil.m_37294_(this, this::m_5603_);
-+      if (hitresult.m_6662_() != HitResult.Type.MISS && !net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, hitresult))
-       this.m_6532_(hitresult);
+-      this.m_6532_(hitresult);
++      if (hitresult.m_6662_() != HitResult.Type.MISS)
++      {
++         switch (net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, hitresult))
++         {
```

Comment:
Formatting: In Minecraft code, the brace style should follow the decompiler output, which is braces on the same line.